tests/encodehelp.h

```diff
@@ -103,6 +130,16 @@ static bool process_cmdline(int argc, char *argv[])
             if (strlen(optarg) == 4)
                 inputFourcc = VA_FOURCC(optarg[0], optarg[1], optarg[2], optarg[3]);
             break;
+        case 1:
```

Comment:
Can you specify why "may conflict with short option" and "Better let getopt_long_only return 0 for long option. and check the option_index to do things" ?